admin_manual/configuration/user/user_provisioning_api.rst

```diff
@@ -466,38 +583,48 @@ XML Output
     <data>
       <element>testgroup</element>
     </data>
-  </ocs>  
-  
+  </ocs>
+
 Instruction Set For Groups
-==========================  
+==========================
 
-**groups / getgroups**
-----------------------
+Get Groups
+----------
 
-Retrieves a list of groups from the ownCloud server. Authentication is done by 
-sending a Basic HTTP Authorization header.
+Retrieves a list of groups from the ownCloud server.
+Authentication is done by sending a Basic HTTP Authorization header.
 
-**Syntax: ocs/v1.php/cloud/groups**
+=========================== ======= ==============
+Request Path                Method  Content Type
+=========================== ======= ==============
+``ocs/v1.php/cloud/groups`` ``GET`` ``text/plain``
+=========================== ======= ==============
 
-* HTTP method: GET
-* url arguments: search - string, optional search string
-* url arguments: limit - int, optional limit value
-* url arguments: offset - int, optional offset value
+======== ====== ======================================
+Argument Type   Description
+======== ====== ======================================
+search   string optional search string
+limit    int    optional limit value
+offset   int    optional offset value
+======== ====== ======================================
 
-Status codes:
+Status Codes
+^^^^^^^^^^^^
 
 * 100 - successful
 
 Example
 ^^^^^^^
 
-  * GET ``http://admin:secret@example.com/ocs/v1.php/cloud/groups?search=adm``
-  * Returns list of groups matching the search string.
+.. code-block:: console
+
+  # Returns list of groups matching the search string.
+  curl http://admin:secret@example.com/ocs/v1.php/cloud/groups?search=adm
```

Comment:
This command returns an empty list, because of the default 4-cahr min search limit. Change to:
``?search=admi``
?

Reply:
Well spotted!